statsmodels/genmod/tests/test_glm_weights.py

```diff
@@ -779,7 +778,7 @@ def test_warnings_raised():
     cov_kwds = {'groups': gid, 'use_correction': False}
 
     # Work around for buggy pytest repeated warning capture on Python 2.7
-    warning_type = SpecificationWarning if PY3 else None
+    warning_type = SpecificationWarning
```

Comment:
comment above unnecessary

Reply:
Should look for "Python 2.7" or just " 2.7" or "2.7 "